src/theme.tsx

```diff
@@ -1,15 +1,15 @@
 const theme = {
   alert: {
     base:
-      "lib:font-sans lib:flex lib:items-center lib:w-full lib:overflow-hidden lib:p-4 lib:bg-blue-50",
```

Comment:
I believe that not using `lib:bg-blue-50` on the base styles is optimal because in the component it will always get fallback to "info" variant which would apply that anyways.

This will prevent eliminate the need of using `bg-blue-50` to override the base styles in the variants

Reply:
Understood, we need to see and fix them.

Need to remove the size & variants from base.